packages/features/bookings/lib/handleNewBooking/test/recurring-event.test.ts

```diff
@@ -486,120 +942,67 @@ describe("handleNewBooking", () => {
-            ],
-          });
+        const assignedUserIds1 = createdBookings1.map((booking) => booking.userId);
 
-          expectBookingCreatedWebhookToHaveBeenFired({
-            booker,
-            organizer,
-            location: "integrations:daily",
-            subscriberUrl: "http://my-webhook.example.com",
-            //FIXME: File a bug - All recurring bookings seem to have the same URL. They should have same CalVideo URL which could mean that future recurring meetings would have already expired by the time they are needed.
-            videoCallUrl: `${WEBAPP_URL}/video/${createdBookings[0].uid}`,
-          });
-        }
+        const createdBookings2 = await handleRecurringEventBooking(req1, res1);
 
-        // expectWorkflowToBeTriggered();
+        const assignedUserIds2 = createdBookings2.map((booking) => booking.userId);
 
-        expectSuccessfulBookingCreationEmails({
-          booker,
-          booking: {
-            // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-            uid: createdBookings[0].uid!,
-            urlOrigin: CAL_URL,
-          },
-          organizer,
-          emails,
-          iCalUID: "MOCKED_GOOGLE_CALENDAR_ICS_ID",
-          recurrence: {
-            ...recurrence,
-            count: recurringCountInRequest,
-          },
-        });
-
-        expect(emails.get().length).toBe(2);
+        const allSameHosts = (assignedUserIds: (number | null | undefined)[]) =>
+          assignedUserIds.every((id) => id === assignedUserIds[0]);
 
-        expectSuccessfulCalendarEventCreationInCalendar(calendarMock, [
-          {
-            calendarId: "event-type-1@google-calendar.com",
-            videoCallUrl: "http://mock-dailyvideo.example.com/meeting-1",
-          },
-          {
-            calendarId: "event-type-1@google-calendar.com",
-            videoCallUrl: "http://mock-dailyvideo.example.com/meeting-1",
-          },
-          {
-            calendarId: "event-type-1@google-calendar.com",
-            videoCallUrl: "http://mock-dailyvideo.example.com/meeting-1",
-          },
-          {
-            calendarId: "event-type-1@google-calendar.com",
-            videoCallUrl: "http://mock-dailyvideo.example.com/meeting-1",
-          },
-        ]);
-      },
-      timeout
-    );
-
-    test(
-      `should create successful bookings for the number of slots requested even if the last slot is already booked as long as first two slots are free
-        1. Should create the same number of bookings as requested slots in the database
-        2. Should send emails for the first booking only to the booker as well as organizer
-        3. Should create a calendar event for every booking in the destination calendar
-        3. Should trigger BOOKING_CREATED webhook for every booking
-    `,
-      async ({ emails }) => {
-        const recurringCountInRequest = 4;
+        expect(createdBookings1[0].userId).not.toBe(createdBookings2[0].userId);
```

Comment:
Both users are available:
- checks if the two separate booked recurring events are assigned to two differnt round robin hosts
- checks if a series of bookings is assigned to the same host